app/services/analytics.rb

```diff
@@ -198,9 +198,9 @@ def browser_attributes
   TELEPHONY_OTP_SENT = 'Telephony: OTP sent'.freeze
   TOTP_SETUP_VISIT = 'TOTP Setup Visited'.freeze
   TOTP_USER_DISABLED = 'TOTP: User Disabled TOTP'.freeze
-  TWILIO_PHONE_VALIDATION_FAILED = 'Twilio Phone Validation Failed'.freeze
-  TWILIO_SMS_INBOUND_MESSAGE_RECEIVED = 'Twilio SMS Inbound Message Received'.freeze
-  TWILIO_SMS_INBOUND_MESSAGE_VALIDATION_FAILED = 'Twilio SMS Inbound Validation Failed'.freeze
+  OTP_PHONE_VALIDATION_FAILED = 'Twilio Phone Validation Failed'.freeze
```

Comment:
😭 too bad we need to leave Twilio in the value for consistency

Reply:
Yeah 😥